libs/admin/src/lib/components/workgroup-edit/workgroup-edit.component.scss

```diff
@@ -0,0 +1,55 @@
+@use "../../styles/variables";
+
+.workgroup-edit {
+  padding: 1.5rem;
+
+  .workgroup-edit__form {
```

Comment:
Remove the `.workgroup-edit__` prefix on all classes in this component as every element in this component's template is wrapped inside the `.workgroup-edit` div anyways.

For the same reasons, you can also move all classes nested within `.workgroup-edit` to the CSS's top level, as nesting them does not change how they are applied anyways.